Framework/API/src/Algorithm.cpp

```diff
@@ -244,18 +244,20 @@ const std::string Algorithm::workspaceMethodInputProperty() const { return ""; }
 
 //---------------------------------------------------------------------------------------------
 /** Initialization method invoked by the framework. This method is responsible
-*  for any bookkeeping of initialization required by the framework itself.
-*  It will in turn invoke the init() method of the derived algorithm,
-*  and of any Child Algorithms which it creates.
-*  @throw runtime_error Thrown if algorithm or Child Algorithm cannot be
-*initialised
-*
-*/
+ *  for any bookkeeping of initialization required by the framework itself.
+ *  It will in turn invoke the init() method of the derived algorithm,
+ *  and of any Child Algorithms which it creates.
+ *  @throw runtime_error Thrown if algorithm or Child Algorithm cannot be
+ *initialised
+ *
+ */
 void Algorithm::initialize() {
   // Bypass the initialization if the algorithm has already been initialized.
   if (m_isInitialized)
     return;
 
+  m_reservedList.clear();
```

Comment:
- What code outside (&before) `init()` would declare index properties?
- If it did, why would you want to clear that vector??